packages/server/lambdas/list-users.ts

```diff
@@ -9,26 +9,19 @@ const database = await DatabaseClient.build();
 export async function handleRequest(event: ALBEvent, _context: Context) {
   try {
     const user = await getUser(event);
-    log.info(user, 'Listing users');
```

Comment:
Return this logging (so it won't be forgotten that easily when real things start to happen.